docs/source/introduction/introduction.md

```diff
@@ -14,13 +14,13 @@ Once you are set up, we suggest working through our examples, including:
 - An [introduction to the project template](../get_started/example_project.md) using the Iris dataset
 - A more detailed [spaceflights tutorial](../tutorial/tutorial_template.md) to give you hands-on experience
 
-We also recommend the [frequently asked questions](../faq/faq.md) and the [API reference documentation](/kedro.rst) for additional information.
+We also recommend the [frequently asked questions](../faq/faq.md) and the [API reference documentation](/kedro) for additional information.
 
 ## Assumptions
 
 We have designed the documentation and the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) for anyone new to Kedro. The more knowledge of Python you have, the easier you will find the learning curve.
 
-```eval_rst
-.. note::  There are a number of excellent online resources for learning Python, but you should choose those that reference Python 3, as Kedro is built for Python 3.6+. There are many curated lists of online resources, such as the `official Python programming language website <https://www.python.org/>`_ and `this list of free programming books and tutorials <https://github.com/EbookFoundation/free-programming-books/blob/master/books/free-programming-books-langs.md#python>`_.
+```{note}
+There are a number of excellent online resources for learning Python, but you should choose those that reference Python 3, as Kedro is built for Python 3.6+. There are many curated lists of online resources, such as the `official Python programming language website <https://www.python.org/>`_ and `this list of free programming books and tutorials <https://github.com/EbookFoundation/free-programming-books/blob/master/books/free-programming-books-langs.md#python>`_.
```

Comment:
I'm afraid all the hyperlinks under `eval_rst` earlier blocks might've been broken now as these are parsed as Markdown now.  :smiling_face_with_tear:
<img width="1122" alt="image" src="https://user-images.githubusercontent.com/11796986/165945602-50120159-2b09-455f-bcc4-16f61145c956.png">

Reply:
Great catch! I will fix them. That's what I get for not listening and scripting the change to `{note}` lol.

Reply:
@avan-sh They should all be resolved, plus caught some more existing errors along the way!